Assets/PatchKit Patcher/Scripts/AppUpdater/Commands/InstallContentCommand.cs

```diff
@@ -143,7 +145,7 @@ public override void Execute(CancellationToken cancellationToken)
                     string nameHash;
                     if (mapHashExtractedFiles.TryGetHash(filePath, out nameHash))
                     {
-                        var sourceFile = new SourceFile(filePath, packageDir.Path, usedSuffix, nameHash, _versionContentSummary.Size);
+                        var sourceFile = new SourceFile(filePath, packageDir.Path, usedSuffix, nameHash, _versionContentSummary.Files[i].Size);
```

Comment:
Is this change related to the thread pool support?

If not please move it to another PR.